include/libdnf/rpm/package_sack.hpp

```diff
@@ -98,27 +98,39 @@ class PackageSack {
         }
     };
 
-    // LoadRepoFlags:
-    // @NONE:                    Load only main solvables
-    // @USE_FILELISTS:           Add the filelists metadata
-    // @USE_PRESTO:              Add presto deltas metadata
-    // @USE_UPDATEINFO:          Add updateinfo metadata
-    // @USE_OTHER:               Add other metadata
     enum class LoadRepoFlags {
-        NONE = 0,
-        USE_FILELISTS = 1 << 1,
-        USE_PRESTO = 1 << 2,
-        USE_UPDATEINFO = 1 << 3,
-        USE_OTHER = 1 << 4,
+        /// Load the primary repodata
```

Comment:
This is public documentation, right? I'd leave out the particle here: _"Load primary repodata"_

Reply:
Fixed.